peer.go

```diff
@@ -884,37 +951,14 @@ func (p *peer) handleLocalClose(req *closeLinkReq) {
 		},
 	}
 
-	// Finally, launch a goroutine which will request to be notified by the
-	// ChainNotifier once the closure transaction obtains a single
-	// confirmation.
 	go func() {
-		// TODO(roasbeef): add param for num needed confs
-		notifier := p.server.chainNotifier
-		confNtfn, err := notifier.RegisterConfirmationsNtfn(closingTxid, 1)
-		if err != nil {
-			req.err <- err
-			return
-		}
+		doneChan := make(chan struct{})
+		go p.waitForCloseConfirmation(channel, closingTxid, doneChan)
```

Comment:
While you're at, this method can also be used within the `rpcserver` when we're handling the force closure of a channel.